frontend/src/components/PhoneNumberInput/PhoneNumberInputContext.tsx

```diff
@@ -131,7 +131,13 @@ const useProvidePhoneNumberInput = ({
       return props.placeholder
     }
 
-    const exampleNumber = getExampleNumber(country, examples)?.formatNational()
+    // Cast to Examples type: JSON files contain plain strings, but the library
+    // expects NationalNumber (tagged type). This is safe at runtime as the library
+    // handles strings correctly.
+    const exampleNumber = getExampleNumber(
```

Comment:
question: wondering if this cast is needed - let me run some checks to see if it works without